internal/cephfs/core/volume.go

```diff
@@ -297,8 +303,8 @@ func (s *subVolumeClient) ResizeVolume(ctx context.Context, bytesQuota int64) er
 	newLocalClusterState(s.clusterID)
 	// resize subvolume when either it's supported, or when corresponding
 	// clusterID key was not present.
-	if clusterAdditionalInfo[s.clusterID].resizeState == unknown ||
-		clusterAdditionalInfo[s.clusterID].resizeState == supported {
+	if clusterAdditionalInfo[s.clusterID].resizeState.Load() == unknown ||
+		clusterAdditionalInfo[s.clusterID].resizeState.Load() == supported {
```

Comment:
This should also go to the helper function like in metadata.go as an enhancement.